cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -309,18 +316,45 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
     });
 
     it("Validates auto tagging of applications based on assessment answers", function () {
-        //automates polarion MTA-387
+        //automates polarion MTA-387 and MTA-502
```

Comment:
This needs to be updated to :
Validates auto tagging of applications and archetypes based on assessment answers

Reply:
done